framework/test/BaseActiveFixture.php

```diff
@@ -115,4 +109,32 @@ public function unload()
         $this->data = [];
         $this->_models = [];
     }
+
+    /**
+     * Returns the fixture data.
+     *
+     * The default implementation will try to return the fixture data by including the external file specified by [[dataFile]].
+     * The file should return the data array that will be stored in [[data]] after inserting into the database.
+     *
+     * @param string $dataFile the data file path
+     * @return array the data to be put into the database
+     * @throws InvalidConfigException if the specified data file does not exist.
+     */
+    protected function loadDataFile($dataFile)
+    {
+        if ($dataFile === false || $dataFile === null) {
+            return [];
+        }
+
+        if ($this->dataFolder !== null) {
+            $dataFile = $this->dataFolder . '/' . basename($dataFile);
```

Comment:
If data file is specified with absolute path then it's meant to be loaded from where it's specified, isn't it?